server/src/controllers/christmas-tree.es6

```diff
@@ -41,45 +36,68 @@ const translateRegulationsFromDatabaseToClient = input => {
           district: location.district,
           allowed: location.allowed,
           type: location.type,
-          description: location.description
+          description: location.description,
+          imageFilename: location.imageFilename
         };
       })
     }
   }; 
 };
 
+christmasTree.getForests = (req, res) => {
 
-christmasTree.getOneRegulations = (req, res) => {
+  treesDb.forests.findAll({
+    attributes: [
+      'id',
+      'forestName',
+      'description'
+    ]
+  }).then(results => {
+    if (results) {
+      res.status(200).json(results);
+    } else {
+      res.status(404).send();
+    }
+  })
+  .catch(error => {
+    res.status(400).json(error);
+  });
+};
+
+
+christmasTree.getOneGuidelines = (req, res) => {
 
-  christmasTreeRegulations.findOne({
+  treesDb.forests.findOne({
     where: {
       id: req.params.id
     },
     include: [
       {
-        model: forestSpecies,
+        model: treesDb.forestSpecies,
         include: [
           {
-            model: species,
+            model: treesDb.species,
             include: [
               {
-                model: speciesNotes
+                model: treesDb.speciesNotes
```

Comment:
again let's combine species notes into species until we actually have a use case to keep them separated.